internal/pkg/util/github/secrets.go

```diff
@@ -0,0 +1,68 @@
+package github
+
+import (
+	"context"
+	crypto_rand "crypto/rand"
+	"encoding/base64"
+	"fmt"
+
+	"github.com/google/go-github/v42/github"
+	"golang.org/x/crypto/nacl/box"
+)
+
+// AddRepoSecret adds a secret to a GitHub repo.
+func (c *Client) AddRepoSecret(secretKey, secretValue string) error {
```

Comment:
@lfbdev @daniel-hutao use this interface to create a secret in a repo.

Reply:
got it